### PR TITLE
Added support for color temperature tile feature

### PR DESCRIPTION
**Pull request by PaoloTK**
## Proposed change

The color temperature tile feature has been added for lights that support CCT control.
chrome_Ysk1iY4zfE
chrome_8Xg8kTDezP

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml
type: tile
entity: light.ceiling_lights
features:
  - type: light-color-temp

```

## Additional information

Something to discuss about the current features implementation is the inability to see the current value for the feature. With brightness being the only stock feature right now this is not a problem, because the brightness percentage is listed under the entity name. We could add dynamic text based on the features enabled for the card? But if a lot of features are added I think it would be verbose. An idea I had was to flash for a few seconds the value set in the feature in place of the current brightness percentage for lights. For example this is what the card would look like after setting the picker for the color temperature:
chrome_f2HtGL4b9d

Thoughts?

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by william-sy** [comment]
I am adding a comment to not let this PR die like `frontend/pull/14697` (which is the same idea)

**Comment by frenck** [comment]
@william-sy Please don't do that. Thanks 👍

File: src/panels/lovelace/tile-features/hui-light-color-temp-tile-feature.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      this.stateObj.attributes.color_temp_kelvin != null
+        ? this.stateObj.attributes.color_temp_kelvin
+        : undefined;
+
+    return html`
+      <div class="container">
+        <ha-control-slider
+          .value=${position}
+          min="1"
+          max="100"
```
Comment (piitaya) [inline src/panels/lovelace/tile-features/hui-light-color-temp-tile-feature.ts:65]:
```suggestion
```

**Review by piitaya — DISMISSED** [review]

**Review by piitaya — APPROVED** [review]

**Comment by PaoloTK** [comment]
> We can merge it without changing the displayed state. For now, user can go to the more info to see the kelvin value. I agree, we should have a way to visualize the kelvin value when dragging. We also have the same issue with the brightness when dragging.
> 
> Also, there is another discussion about letting the user choose what to display as state #16661.
> 
> I will look if we can improve the gradient by using the min and max kelvin values like we have in the more info light.

Fair enough, thank you!

**Pull request merged**